### DISCUSSION TITLE
Large event in aws custom client

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#2901] [General Questions & Discussion] [answered]
Hello 

What is your experience for very large event with aws cloud server . Have you try to have More than 25 users per room ? 
What is the max user in the same room you have reach and with wich server ? 
Thx for your reply

**Comment by vjFred** [comment]
have tested on enterprise-cloud multiserver, and i am not happy

**Comment by emclaren** [comment, marked as answer]
Hi there, thanks for reaching out. We’ve had community members host events for thousands of people on AWS. 

Using AWS doesn’t change the constraints about the maximum number of users in a room. There is more information on maximum capacity here: https://github.com/mozilla/hubs/discussions/2680 - generally, we recommend a max of 25 users per room, but you can increase this (this might lead to performance decreases, depending on how powerful your user's devices are).

To estimate the size of the server needed for a large event you can see our documentation on estimating ccu limits https://hubs.mozilla.com/docs/hubs-cloud-aws-estimated-ccu-limits.html 

If you have specific questions about your event or supporting a specific number of users feel free to ask in another Discussion thread. It’s difficult to give more detailed feedback without knowing more details about what you are planning.
